doc/todo/RELEASE.md

```diff
@@ -17,14 +17,15 @@ If no release critical problems are found during testing, continue.
 - Update `doc/COMPILE.md` to reflect actually tested setups
 - Run `scripts/dev/update-infos-status` with arguments for heuristics (--auto) and check
   transitions in plugin status: especially from/to experimental
-- Write release notes (without hash sums, guid or proper download links)
-- `cp doc/todo/NEWS.md doc/news/_preparation_next_release.md`
+- Write release notes with guid (but without giving a final name, hashsums, statistics, proper download links or replacing the `<<VERSION>>` placeholder)
+  > NOTE: hashsums and statistics get automatically inserted into release notes through the release pipeline
 
 ### Increment Version Number
 
 - `CMakeLists.txt`
 - Increment version in output of `doc/tutorials/hello-elektra.md`
 - Increment <Version> of libelektra in `examples/external/java/read-keys-example/pom.xml` and `src/bindings/jna/README.md`
+- Increment the Elektra version in `scripts/docker/alpine/*/release.Dockerfile`
```

Comment:
Does this really work now? Before this PR it did not, because it depends on the release tarball which is added at a later stage.

Reply:
Yes it should work now, because the release tarball gets published before the alpine image is built.